<launch>

    <node name="image_view_stereo" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/stereo_driver_node/image_raw"/>
        <param name="autosize" value="false" />
    </node>

    <node name="image_view_pico" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/pico_flexx/image_depth"/>
        <param name="autosize" value="false" />
    </node>

</launch>
